src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/OffenderAlias.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
+
+import java.time.LocalDate
+
+data class OffenderAlias(
```

Comment:
Tiny thing but I would call this `Alias` instead as that's the name used in the Prison API.

![image](https://user-images.githubusercontent.com/42817036/217903841-7e4d1ea0-4dae-444b-95d5-d2580f707769.png)

Reply:
Good spot. All for consistent naming of how they have things on their end